src/viewport.tsx

```diff
@@ -0,0 +1,121 @@
+import React, {
+  ReactNode,
+  useState,
+  useCallback,
+  useRef,
+  useEffect
+} from 'react'
+import { DebouncedFunc } from 'lodash'
+import lodashThrottle from 'lodash/throttle'
```

Comment:
Just wanted to sanity check that we weren't causing a yuge bundle by using lodash-es.

I'm out of the loop for the typing, but is `DebouncedFunc` a type exported from lodash that can be used for throttled/debounced functions created by lodash?

Reply:
Good questions! yes `DebouncedFunc` is a declaration which I'm using to specify a return type. The import is actually coming from the types packages `@types.lodash` so should get stripped.

TSDX handles optimisations https://github.com/formium/tsdx#using-lodash and bundle analysis seems to be ok:

![image](https://user-images.githubusercontent.com/11708259/94548196-44cb3100-0248-11eb-9e2c-89e1cc164298.png)